### ISSUE TITLE
Add fall, lava, fire damage

### ISSUE THREAD

**Issue by numberZero** [sapier/animals_modpack#116]
Currently a chicken, for example, can fall from 30m on stone, then walk through lava, and take a rest in fire. It doesn't take any damage! (But disappears just touching water. And swords work, of course)

**Comment by sapier** [comment]
true, fall damage doesn't exist atm I'm gonna see when I've time to implement it.
Lava is already supposed to kill it, I'll fix this in next version.

**Comment by kilbith** [comment]
Hey @sapier, come back into MT's stuff please. We miss you.
